packages/cbioportal-frontend-commons/src/components/signal/SignalHelper.tsx

```diff
@@ -176,10 +176,10 @@ export const FREQUENCY_COLUMNS_DEFINITION = {
                 0
             );
             const frequency =
```

Comment:
this is actually percentage not frequency, right? maybe rename `frequency` as `percent`?